### ISSUE TITLE
Missing dependancy on qtdeclarative5-qtquick2-plugin

### ISSUE THREAD

**Issue by blutack** [diydrones/apm_planner#532]
After installing the 2.0.15 on a fully updated Ubuntu Trusty, it failed to launch with the following error.

```
DEBUG 2014-11-26T17:32:04.579  QUrl( "file:///usr/share/APMPlanner2/qml/ApmToolBar.qml" )  
file:///usr/share/APMPlanner2/qml/ApmToolBar.qml:17:1: module "QtQuick" is not installed 
     import QtQuick 2.0 
     ^ 
DEBUG 2014-11-26T17:32:04.581 QML Status: 3 
Segmentation fault (core dumped)
```

I fixed this by installing the package qtdeclarative5-qtquick2-plugin. It would be worth adding this as a dependancy of the apmplanner package.

**Comment by hugokoopmans** [comment]
this issue is still there on linux mint Petra

**Comment by blutack** [comment]
Hugo, the current 2.0.15 packages are based on a release from November so unless there was an interim package release without a version bump this fix will not make it in until 2.0.16. Malcolm can correct me if I'm wrong. In the meantime, you can manually apply the fix by installing the package.

**Comment by gmorph** [comment]
I had the package installed and it doesn't help.  The issue has been resolved by this fix I believe:
https://github.com/diydrones/apm_planner/pull/622
but it needs to be merged into Master.  It will be done shortly I'm sure.
